Add french translation file

Arona74 opened the pull request:
Hi, just made a french lang file for last 1.21 version.

Arona74 committed:
Create fr_fr.json

Add french translation

Arona74 committed:
Update fr_fr.json

Updated using last en_us.json file from 1.21 branch

Fooshcookie commented:
I'll check with the big man.

elliotfontaine commented:
Très très fan des traductions que tu as proposé @Arona74 🤩

Pour **Gwib** et **Gwibling**, est-ce qu'on pourrait pas trouver un nom bizarre, plutôt que Silure Albinos qui évoque un animal réel ?
- Siluvore
- Abyssure
- Pâluron

Et pour **Rotbeast**, **Rotling**, **Rotwalker**, plutôt que "pourri" j'ai pensé aux adjectifs suivants:
- putride
- putréfié
- charogné

J'espère que ce PR finira par être mergé!

Arona74 commented:
Merci du retour!L’auteur du mod utilise beaucoup de mots venant de l’argot anglais voir australien, très souvent connoté vulgaire. Par exemple le gwib: http://gwib.urbanup.com/2842520Donc faut choisir entre une traduction se voulant plus réelle ou connotée mais c’est jamais simple de trouver.Pour les « rots », j’hésitais entre pourri/pourriture et putréfié/putréfaction. 6 mois plus tard j’ai une préférence pour putréfié. Je pense que j’étais allé sur pourri pour le côté plus sale et connoté du mot.Le 11 févr. 2025 à 03:42, Elliot Fontaine ***@***.***> a écrit :﻿
Très très fan des traductions que tu as proposé @Arona74 🤩
Pour Gwib et Gwibling, est-ce qu'on pourrait pas trouver un nom bizarre, plutôt que Silure Albinos qui évoque un animal réel ?

Siluvore
Abyssure
Pâluron

Et pour Rotbeast, Rotling, Rotwalker, plutôt que "pourri" j'ai pensé aux adjectifs suivants:

putride
putréfié
charogné

J'espère que ce PR finira par être mergé!

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "block.undergarden.carved_gloomgourd": "Lugourde sculptée",
+  "block.undergarden.chiseled_depthrock_bricks": "Roche des profondeurs taillée",
+  "block.undergarden.chiseled_shiverstone_bricks": "Pierre givrée taillée sculptée",
+  "block.undergarden.chiseled_tremblecrust_bricks": "Chiseled Tremblecrust Bricks",
```
Not localized.

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "block.undergarden.coarse_deepsoil": "Terre des profondeurs crue",
+  "block.undergarden.cracked_depthrock_bricks": "Roche des profondeurs taillée craquelée",
+  "block.undergarden.cracked_shiverstone_bricks": "Pierre givrée  taillée craquelée",
+  "block.undergarden.cracked_tremblecrust_bricks": "Cracked Tremblecrust Bricks",
```
Not localized.

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "block.undergarden.stripped_wigglewood_wood": "Bois de Bois tordu écorcé",
+  "block.undergarden.tall_deepturf": "Grande Herbe des profondeurs",
+  "block.undergarden.tall_shimmerweed": "Grande Herbe luisante",
+  "block.undergarden.tremblecrust": "Tremblecrust",
```
Not localized (from line 232 to 242).

elliotfontaine reviewed (commented):
EN: You missed some lines that are still in english.
FR: Tu as oublié de traduire quelques lignes.

Arona74 committed:
Update fr_fr.json

Add Tremblecrust translation
Change Gwib and Gwibling translation
Minor typo changes

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "advancement.undergarden.cloggrum_battleaxe.desc": "Obtenez une Hache de guerre en Cloggrum sur un Oublié en portant une.",
+  "advancement.undergarden.cloggrum_battleaxe.title": "Mains Froides et Mortes",
+  "advancement.undergarden.enter_undergarden.desc": "La terre oubliée vous attend...",
+  "advancement.undergarden.enter_undergarden.title": "Entrez dans le Jardin Souterrain",
```
```suggestion
  "advancement.undergarden.enter_undergarden.title": "Entrez dans l'Undergarden",
```

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "advancement.undergarden.kill_rotling.title": "Contraception",
+  "advancement.undergarden.kill_scintling.desc": "Vous avez tué un Scintillant innocent. Vous êtes un monstre...",
+  "advancement.undergarden.kill_scintling.title": "Terrible Personne",
+  "advancement.undergarden.mine_ore.desc": "Acquérir n'importe quel minerai d'Undergarden.",
```
```suggestion
  "advancement.undergarden.mine_ore.desc": "Obtenir n'importe quel minerai de l'Undergarden.",
```

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "block.undergarden.tremblecrust_utherium_ore": "Tremblecrust Utherium Ore",
+  "block.undergarden.tremblecrust_wall": "Tremblecrust Wall",
+  "block.undergarden.underbean_bush": "Buisson d'Haricots des profondeurs",
+  "block.undergarden.undergarden_portal": "Portail d'Undergarden",
```
```suggestion
  "block.undergarden.undergarden_portal": "Portail de l'Undergarden",
```

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "subtitles.item.slingshot.gronglet_shoot": "Tir de Lance-pierre",
+  "subtitles.item.slingshot.shoot": "Tir de Lance-pierre",
+  "tooltip.undergarden.cloggrum_boots": "Bave de Scintillant ne vous ralentit plus lorsque portées.",
+  "tooltip.undergarden.forgotten_tool": "Mine les blocs d'Undergarden 1.5x plus vite.",
```
```suggestion
  "tooltip.undergarden.forgotten_tool": "Mine les blocs de l'Undergarden 50% plus vite.",
```

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "subtitles.item.slingshot.shoot": "Tir de Lance-pierre",
+  "tooltip.undergarden.cloggrum_boots": "Bave de Scintillant ne vous ralentit plus lorsque portées.",
+  "tooltip.undergarden.forgotten_tool": "Mine les blocs d'Undergarden 1.5x plus vite.",
+  "tooltip.undergarden.forgotten_weapon": "Inflige 1.5x de dégâts aux entités d'Undergarden qui ne sont pas des boss.",
```
```suggestion
  "tooltip.undergarden.forgotten_weapon": "Inflige 50% de dégâts supplémentaires aux entités de l'Undergarden qui ne sont pas des boss.",
```

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "block.undergarden.wigglewood_stairs": "Escalier en Bois tordu",
+  "block.undergarden.wigglewood_trapdoor": "Trappe en Bois tordu",
+  "block.undergarden.wigglewood_wood": "Bois de Bois tordu",
+  "config.undergarden.return_portal_frame_block_id": "Return Portal Frame Block ID",
```
```suggestion
  "config.undergarden.return_portal_frame_block_id": "ID de bloc pour le portail retour",
```

elliotfontaine replied:
Mince j'ai fait mes suggestions sur le commit précédent au lieu du plus récent...

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "block.undergarden.wigglewood_trapdoor": "Trappe en Bois tordu",
+  "block.undergarden.wigglewood_wood": "Bois de Bois tordu",
+  "config.undergarden.return_portal_frame_block_id": "Return Portal Frame Block ID",
+  "config.undergarden.toggle_undergarden_fog": "Toggle Undergarden Fog",																		
```
```suggestion
  "config.undergarden.toggle_undergarden_fog": "Brouillard dans l'Undergarden",					
```

elliotfontaine commented on src/main/resources/assets/undergarden/lang/fr_fr.json:
```diff
@@ -0,0 +1,544 @@
+  "tooltip.undergarden.forgotten_weapon": "Inflige 1.5x de dégâts aux entités d'Undergarden qui ne sont pas des boss.",
+  "tooltip.undergarden.froststeel_weapon": "Ralentit les cibles.",
+  "tooltip.undergarden.slingshot_ammo": "Peut être utilisé comme munitions du Lance-pierre.",
+  "tooltip.undergarden.utherium_weapon": "Inflige 1.5x de dégâts aux Engeances de la pourriture.",
```
```suggestion
  "tooltip.undergarden.utherium_weapon": "Inflige 50% de dégâts supplémentaires aux Engeances de la pourriture.",
```

elliotfontaine reviewed (changes requested):
- Traduction des paramètres.
- `50% [...] supplémentaires` plutôt que `1.5x`
- `de l'Undergarden` plutôt que `d'Undergarden`

Arona74 committed:
Update src/main/resources/assets/undergarden/lang/fr_fr.json

Co-authored-by: Elliot Fontaine <92150839+elliotfontaine@users.noreply.github.com>

Arona74 committed:
Update src/main/resources/assets/undergarden/lang/fr_fr.json

Co-authored-by: Elliot Fontaine <92150839+elliotfontaine@users.noreply.github.com>

Arona74 committed:
Update src/main/resources/assets/undergarden/lang/fr_fr.json

Co-authored-by: Elliot Fontaine <92150839+elliotfontaine@users.noreply.github.com>

Arona74 committed:
Update src/main/resources/assets/undergarden/lang/fr_fr.json

Co-authored-by: Elliot Fontaine <92150839+elliotfontaine@users.noreply.github.com>

Arona74 committed:
Update src/main/resources/assets/undergarden/lang/fr_fr.json

Co-authored-by: Elliot Fontaine <92150839+elliotfontaine@users.noreply.github.com>

Arona74 committed:
Update fr_fr.json

Add elliotfontaine suggestions

Arona74 commented:
EN:
Added elliotfontaine suggestions

FR:
J'ai ajouté tes suggestions.

elliotfontaine commented:
@quek04 Hi! This is ready to be merged. 👍
